constance/backends/database/apps.py

```diff
@@ -0,0 +1,8 @@
+from django.apps import AppConfig
+from django.utils.translation import ugettext_lazy as _
+
+
+class ConstanceConfig(AppConfig):
+    name = 'constance.backends.database'
+    label = 'constance_db'
```

Comment:
See my comments below, please don't rename the label of the app since it'll break the already applied migrations of deployed versions of the app